src/app/core/guards/onboarding.guard.ts

```
@@ -8,18 +8,23 @@
+    _: ActivatedRouteSnapshot,
+    _: RouterStateSnapshot
   ): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
     return this.spenderOnboardingService.checkForRedirectionToOnboarding().pipe(
       map((shouldRedirectToOnboarding) => {
         if (shouldRedirectToOnboarding) {
+          this.router.navigate(['/', 'enterprise', 'spender_onboarding']);
           return false;
         }
         return true;
+      }),
+      catchError((error) => {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)